apps/address-book/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@aragon/api": "2.0.0-beta.8",
     "@aragon/api-react": "2.0.0-beta.6",
-    "@aragon/ui": "^1.0.0-alpha.26",
+    "@aragon/ui": "^1.0.0",
```

Comment:
I think it is important to use the same version, as the `^` character allows for every version in the range between `0.9.9` and `2.0.0` (not included), so in this case, `lerna` decides not to hoist the `ui` package for this app:

```
lerna WARN EHOIST_PKG_VERSION "@tps/apps-address-book" package depends on @aragon/ui@^1.0.0, which differs from the hoisted @aragon/ui@1.0.0.
lerna WARN EHOIST_PKG_VERSION "@tps/apps-discussions" package depends on @aragon/ui@1.0.0-alpha.11, which differs from the hoisted @aragon/ui@1.0.0.
```

I would also update discussions `ui`

Reply:
Basically: pin it in all apps, or not pin it all